package.json

```diff
@@ -62,14 +63,19 @@
 		"build-css:editor": "check-npm-client && node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client client/assets/stylesheets/editor.scss public/editor.css --output-style compressed && yarn run -s postcss public/editor.css",
 		"build-css:reader-mobile": "check-npm-client && node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client client/assets/stylesheets/reader-mobile.scss public/reader-mobile.css --output-style compressed && yarn run -s postcss public/reader-mobile.css",
 		"build-devdocs:search-index": "check-npm-client && node bin/generate-devdocs-search-index.js",
+		"build-desktop": "check-npm-client && run-s build-desktop:config build-desktop:server build-desktop:client build-desktop:app",
```

Comment:
There should be an additional check to ensure that `secrets.json` exists and is valid before proceeding to build the app, as with the original Makefile logic ([link](https://github.com/Automattic/wp-desktop/blob/develop/Makefile#L134-L145)) to mitigate the risk of an invalid app being built either locally or in CI.

Reply:
Added the check in the latest commit. It's now in the script that runs during the `build-desktop:config` task.